tilequeue/wof.py

```diff
@@ -66,33 +75,194 @@ def fetch_wof_neighbourhoods(url):
         except ValueError:
             continue
 
-        x, y = pyproj.transform(latlng_proj, merc_proj, lng, lat)
+        file_hash = row[hash_idx]
+
+        label_x, label_y = pyproj.transform(latlng_proj, merc_proj, lng, lat)
+        label_position = shapely.geometry.Point(label_x, label_y)
 
-        neighbourhood = Neighbourhood(wof_id, name, x, y)
-        yield neighbourhood
+        neighbourhood_meta = NeighbourhoodMeta(
+            wof_id, name, label_position, file_hash)
+        yield neighbourhood_meta
 
 
-class WofNeighbourhoodFetcher(object):
+def fetch_wof_url_meta_neighbourhoods(url):
+    r = requests.get(url, stream=True)
```

Comment:
Added the assertion in 0ad66eb.

Looks like `requests` does have support for [retrying](http://docs.python-requests.org/en/latest/api/#requests.adapters.HTTPAdapter) out of the box, but this wouldn't help with the 503s. One idea is that we could try creating a custom http adapter that would consider 503 responses as a signal to sleep and retry again.

But maybe it's better to just punt on this for now and wait if we see this kind of problem in practice?